docs/source/hooks/introduction.md

```diff
@@ -32,6 +32,8 @@ The naming convention for error hooks is `on_<noun>_error`, in which:
 
 [kedro.framework.hooks](/kedro.framework.hooks) lists the full specifications for which you can inject additional behaviours by providing an implementation.
 
+This diagram illustrates the execution order of hooks during `kedro run`:
+![kedro run hook execution order](../../source/meta/images/kedro_run_lifecycle.png)
```

Comment:
Can we remove the reference to `source`?

```suggestion
![kedro run hook execution order](../meta/images/kedro_run_lifecycle.png)
```

Reply:
Also @noklam if you did this with Mermaid, can you paste the code in a comment? Similar to how we're doing it in other diagrams, see for example 

https://github.com/kedro-org/kedro/blob/d433ff31542ee03724930e2f409e9fd5d9819a1e/docs/source/get_started/new_project.md?plain=1#L170-L187